### PR TITLE
KHAYAAL-2-leader description component created

### PR DESCRIPTION
**Pull request by preethisun0801**
leader description component created

### CODE REVIEW

File: src/Components/OurTeams/LeaderDescription.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import "./LeaderDescription.css"
+const LeaderDescription = ({name,position,desc}) => {
```
Comment (roshan1999) [inline src/Components/OurTeams/LeaderDescription.jsx:2]: take input as props and use further.

File: src/Components/OurTeams/LeaderDescription.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    return (
+        <div className="leaderDesc">
+            <h2 className="leaderName">{name}</h2>
```
Comment (roshan1999) [inline src/Components/OurTeams/LeaderDescription.jsx:5]: Use props.nameText, props.positionText, props.descriptionText

File: src/Components/OurTeams/LeaderDescription.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
+* {
+    font-family: -apple-system, BlinkMacSystemFont, "Roboto", sans-serif;
+}
+.leaderDesc {
+    font-weight: 700;
+    line-height: 38px;
+}
+.leaderName {
+    font-size: 38px;
```
Comment (roshan1999) [inline src/Components/OurTeams/LeaderDescription.css:9]: font size should change with screen size. Refer home page to see how font changes with screen size

File: src/Components/OurTeams/LeaderDescription.css [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    color: #2e244b;
+}
+.leaderPosition {
+    font-size:21px;
```
Comment (roshan1999) [inline src/Components/OurTeams/LeaderDescription.css:13]: font size should change with screen size. Refer home page to see how font changes with screen size

File: src/Components/OurTeams/LeaderDescription.css [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    color:#885F88;
+}
+.leaderContents {
+    font-style: italic;
+    font-size: 15px;
```
Comment (roshan1999) [inline src/Components/OurTeams/LeaderDescription.css:18]: font size should change with screen size. Refer home page to see how font changes with screen size

File: src/Components/OurTeams/LeaderDescription.css [hunk 6]
```diff
@@ -0,0 +1,21 @@
+* {
+    font-family: -apple-system, BlinkMacSystemFont, "Roboto", sans-serif;
+}
+.leaderDesc {
+    font-weight: 700;
+    line-height: 38px;
+}
+.leaderName {
+    font-size: 38px;
+    color: #2e244b;
```
Comment (roshan1999) [inline src/Components/OurTeams/LeaderDescription.css:10]: No hardcoding of color. Refer styles/colors.css for the color. Take the closest color to it from the design. 
Refer homepage on how to include the color from colors.css to here

File: src/Components/OurTeams/LeaderDescription.css [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+}
+.leaderPosition {
+    font-size:21px;
+    color:#885F88;
```
Comment (roshan1999) [inline src/Components/OurTeams/LeaderDescription.css:14]: No hardcoding of color. Refer styles/colors.css for the color. Take the closest color to it from the design.
Refer homepage on how to include the color from colors.css to here

File: src/Components/OurTeams/LeaderDescription.css [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+}
+.leaderContents {
+    font-style: italic;
+    font-size: 15px;
+    color:#686868;
```
Comment (roshan1999) [inline src/Components/OurTeams/LeaderDescription.css:19]: No hardcoding of color. Refer styles/colors.css for the color. Take the closest color to it from the design.
Refer homepage on how to include the color from colors.css to here

File: src/Components/OurTeams/LeaderDescription.css [hunk 9]
```diff
@@ -0,0 +1,21 @@
+* {
+    font-family: -apple-system, BlinkMacSystemFont, "Roboto", sans-serif;
+}
+.leaderDesc {
+    font-weight: 700;
+    line-height: 38px;
```
Comment (roshan1999) [inline src/Components/OurTeams/LeaderDescription.css:6]: px -> make note, to make it look good when different screen sizes are used

**Review by roshan1999 — CHANGES_REQUESTED** [review]
Make note of all screen sizes and px will work only for one screen size and will be too small for other screen sizes

**Comment by preethisun0801** [comment]
props changed, fonts changed from px to rem, colors changed from hardcoded to var from colors.css

**Comment by preethisun0801** [comment]
[KHAYAAL-2.docx](https://github.com/roshan1999/interns-khayaal-app/files/8771472/KHAYAAL-2.docx)

**Review by roshan1999 — APPROVED** [review]

**Pull request merged**